src/main/java/com/hackerrank/weather/model/GenresWrapperClass.java

```diff
@@ -0,0 +1,29 @@
+package com.hackerrank.weather.model;
+
+import java.util.List;
+
+public class GenresWrapperClass {
```

Comment:
Use Lombok everywhere. 

*WrapperClass is not a good name. Think of the purpose of this class - is it a Response, or just a DTO or what?